From eba98f0c0bfce6e1061052ed6356c68078225373 Mon Sep 17 00:00:00 2001
From: Herman van Hazendonk <github.com@herrie.org>
Date: Fri, 29 Sep 2017 12:15:00 +0200
Subject: [PATCH] Patch to avoid spamming of log with "Failed to set invocation
 ID..."

Since xattrs on cgroup is not supported in our 3.4 device kernels we get a lot of the following warnings in our logs:

"Failed to set invocation ID on control group ..."

Signed-off-by: Herman van Hazendonk <github.com@herrie.org>
---
 src/core/cgroup.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/core/cgroup.c b/src/core/cgroup.c
index 23a92f9..bb99c44 100644
--- a/src/core/cgroup.c
+++ b/src/core/cgroup.c
@@ -1376,7 +1376,7 @@ static void cgroup_xattr_apply(Unit *u) {
                          sd_id128_to_string(u->invocation_id, ids), 32,
                          0);
         if (r < 0)
-                log_unit_warning_errno(u, r, "Failed to set invocation ID on control group %s, ignoring: %m", u->cgroup_path);
+                log_unit_debug_errno(u, r, "Failed to set invocation ID on control group %s, ignoring: %m", u->cgroup_path);
 }
 
 static bool unit_has_mask_realized(Unit *u, CGroupMask target_mask, CGroupMask enable_mask) {
